### PR TITLE
rename to form-classifier

### PR DESCRIPTION
**Pull request by NPann**
@dpark6060 can you take a look to this one ? I am also going to move this repo to form-classifier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,16 +26,16 @@ input.
 #### Metadata
 
 In general, since fw-classification acts on input metadata, the input file needs to have
-it's metadata populated before running file-classifier. The metadata can live in a few
+it's metadata populated before running form-classifier. The metadata can live in a few
 places depending on how the file will be classified.  The most common would be in the
-`file.info.header.<file-type>` which will be populated by `file-metadata-importer`.  But
+`file.info.header.<file-type>` which will be populated by `form-importer`.  But
 the metadata can also be in a separate file such as the `sidecar.json` for NIfTIs, or in
```
Comment (dpark6060) [inline README.md]: Should remove reference to Nifti.  Also, I'm not certain this can actually save json sidecars...

File: README.md [hunk 2]
```diff
@@ -35,16 +26,16 @@ input.
...
 the hierarchy such as acquisition label, file name, or custom information on any parent
 container.
 
 #### Profile
 
-file-classifier ships with [default
+form-classifier ships with [default
 profiles](https://gitlab.com/flywheel-io/scientific-solutions/lib/fw-classification-profiles/-/tree/main/profiles)
```
Comment (dpark6060) [inline README.md]: not true anymore,  uses custom profile on this repo

**Review by dpark6060 — CHANGES_REQUESTED** [review]
two small things.  I could just change them myself if you'd like

File: README.md (hunk 1 above)
Comment (NPann) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (NPann) [reply README.md]: fixed

**Review by dpark6060 — APPROVED** [review]

**Pull request merged**